### ISSUE TITLE
tensorboard projector mode with custom metadata_header with only one label name

### ISSUE THREAD

**Issue by dizcza** [pytorch/pytorch#38564]
## 🐛 Bug

`.add_embeddings(metadata_header=['some-name'])` breaks the tensorboard to parse the metadata. Reaching http://localhost:6006/#projector shows the error

```
Error merging metadata
Number of tensors (20) do not match the number of lines in metadata (21). Single column metadata should not have a header row.
```

![Bildschirmfoto vom 2020-05-15 20-23-22](https://user-images.githubusercontent.com/7688337/82083388-f3cbfd80-96e9-11ea-9f0a-c5e11a11c05c.png)

## To Reproduce

```python
import torch
from torch.utils.tensorboard import SummaryWriter
mat = torch.randn(20, 10)
units = torch.arange(20).unsqueeze(1).tolist()
writer = SummaryWriter('runs/example')
writer.add_embedding(mat, units, metadata_header=['unit'])
writer.close()
```
Then navigate to localhost:6006 in a browser.

Note: using the default `metadata_header` does not break tensorboard to parse the metadata file.

## Expected behavior

A browser opens Projector tab and shows the data in PCA space.

## Environment

PyTorch version: 1.5.0
Is debug build: No
CUDA used to build PyTorch: None

OS: Ubuntu 18.04.4 LTS
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
CMake version: version 3.10.2

Python version: 3.8
Is CUDA available: No
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA

Versions of relevant libraries:
[pip] numpy==1.18.1
[pip] torch==1.5.0
[pip] torchvision==0.6.0a0+82fd1c8
[conda] blas                      1.0                         mkl  
[conda] cpuonly                   1.0                           0    pytorch
[conda] mkl                       2020.1                      217  
[conda] mkl-service               2.3.0            py38he904b0f_0  
[conda] mkl_fft                   1.0.15           py38ha843d7b_0  
[conda] mkl_random                1.1.0            py38h962f231_0  
[conda] numpy                     1.18.1           py38h4f9e942_0  
[conda] numpy-base                1.18.1           py38hde5b4d6_1  
[conda] pytorch                   1.5.0               py3.8_cpu_0  [cpuonly]  pytorch
[conda] torchvision               0.6.0                  py38_cpu  [cpuonly]  pytorch

## Additional context

If it's a bug on tensorflow/tensorboard side, let me know to move the issue there.

The `metadata.tsv` file has 21 lines because it starts with the label name:

```
unit
0
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
```
